cli/tests/data/dcos.toml

```diff
@@ -1,5 +1,5 @@
 [core]
+ssl_verify = "false"
 timeout = 5
 reporting = false
 dcos_url = "http://dcos.snakeoil.mesosphere.com"
-ssl_verify = "false"
```

Comment:
hmm... something in our test runs must modify this.  I didn't explicitly change it.  I'll fix it.